packages/ContentsGroupingExtensions/src/pages/ContentsGroupingExtension.vue

```diff
@@ -125,17 +123,50 @@ export default {
         sortByExtOrderNumber(extA, extB) {
             return extB.ext_order_no - extA.ext_order_no;
         },
+        loadScript(src) {
```

Comment:
Is this required to use async/await?

Reply:
@yabe-diverta 

We only need to load the core's manifest and core's vendor js first. 
Child components will be loaded asynchronously afterwards with the following 🤔 
```
await Promise.all(
                coreComponents.map((component) =>
                    this.loadScript(prefixUrl + manifest['common/components/extensions/Ext' + component + '.js'])
                )
            );
```

Sorry, is that what you mean?